web/src/styles/global.css

```diff
@@ -2,6 +2,21 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer utilities {
+    .heading-accent {
+        position: relative;
+    }
+    .heading-accent:after {
+        content: '';
+        height: 5px;
+        width: 50px;
+        background-color: #acc9e8;
+        position: absolute;
+        top: 86px;
```

Comment:
I'm not sure this is going to work. What happens if the heading breaks onto a new line?